content/en/docs/concepts/workloads/controllers/replicationcontroller.md

```diff
@@ -284,6 +284,11 @@ machine-level function, such as machine monitoring or machine logging.  These po
 to a machine lifetime: the pod needs to be running on the machine before other pods start, and are
 safe to terminate when the machine is otherwise ready to be rebooted/shutdown.
 
-## For more information
-
-Read [Run Stateless Application Deployment](/docs/tasks/run-application/run-stateless-application-deployment/).
+## {{% heading "whatsnext" %}}
+
+* Learn about [Pods](/docs/concepts/workloads/pods).
+* Learn about [Deployment](/docs/concepts/workloads/controllers/deployment/), the replacement
+  for ReplicationController.
```

Comment:
```suggestion
  for `ReplicationController`.
```

Or possibly 

```suggestion
  for {{< api-reference page="workload-resources/replication-controller-v1" >}}.
```

Reply:
I wouldn't make either of these changes @RinkiyaKeDad: I don't think they'd improve the page from what I've drafted.